src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/SemanticTokensTest.cs

```diff
@@ -1006,12 +1006,14 @@ private async Task VerifySemanticTokensAsync(string documentText, bool precise,
             options.HtmlVirtualDocumentSuffix == "__virtual.html",
             MockBehavior.Strict);
 
+        var cSharpSemanticTokensProvider = new LSPCSharpSemanticTokensProvider(_clientConnection.Object, LoggerFactory);
```

Comment:
>cShar [](http://example.com/codeflow?start=12&length=5)

super-nit :) funky capitalization, seems like we use "csharpXXX" elsewhere

Reply:
100%! I was chasing the code fixes for a bunch of these, but I guess I missed a few. Thanks!